### ISSUE TITLE
Automatic start on login

### ISSUE THREAD

**Issue by trules** [keybase/client#3190]
Keybase GUI Version: 1.0.16-20160621190121+4d16a4b
I am unable to find an option to disable automatic start on login.

**Comment by gardiner** [comment]
Yes this is really annoying. Especially because you have to login/enter your passphrase again.
